File: HostAndGuest_JavProject/src/com/hostandguest/entities/Property.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package com.hostandguest.entities;


import java.util.ArrayList;
import java.util.Date;
import java.util.List;

/**
 *
 * @author The
 */
public class Property {
    private int id;
    private int nbRooms;
    private int price;
    private Date publicationDate;
    private String description;
    private List<Object> equipements;
    private List<Object> imagesPath;
    private String location;
    private boolean reported = false;
    private boolean enabled = true;
    
    // Many To One Relationship
    private int host_id;

    public Property() {
        this.publicationDate = new Date();
        equipements = new ArrayList<>();
        imagesPath = new ArrayList<>();
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public int getNbRooms() {
        return nbRooms;
    }

     public List<Object> getEquipements() {
        return equipements;
    }

    public void setEquipements(List<Object> equipements) {
        this.equipements = equipements;
    }
    
    public void setNbRooms(int nbRooms) {
        this.nbRooms = nbRooms;
    }

    public int getPrice() {
        return price;
    }

    public void setPrice(int price) {
        this.price = price;
    }

    public Date getPublicationDate() {
        return publicationDate;
    }

    public void setPublicationDate(Date publicationDate) {
        this.publicationDate = publicationDate;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public List<Object> getImagesPath() {
        return imagesPath;
    }

    public void setImagesPath(List<Object> imagesPath) {
        this.imagesPath = imagesPath;
    }



    public String getLocation() {
        return location;
    }

    public void setLocation(String location) {
        this.location = location;
    }

    public boolean isReported() {
        return reported;
    }

    public void setReported(boolean reported) {
        this.reported = reported;
    }

    public boolean isEnabled() {
        return enabled;
    }

    public void setEnabled(boolean enabled) {
        this.enabled = enabled;
    }

    public int getHost_id() {
        return host_id;
    }

    public void setHost_id(int host_id) {
        this.host_id = host_id;
    }

    @Override
    public String toString() {
        return "Property{" + "id=" + id + ", nbRooms=" + nbRooms + ", price=" + price + ", publicationDate=" + publicationDate + ", description=" + description + ", equipements=" + equipements + ", imagesPath=" + imagesPath + ", location=" + location + ", reported=" + reported + ", enabled=" + enabled + ", host_id=" + host_id + '}';
    }
    
}

File: Host_n_Guest/src/PropertyBundle/Entity/Property.php
<?php

namespace PropertyBundle\Entity;

use DateTime;
use Doctrine\ORM\Mapping as ORM;
use Eko\FeedBundle\Item\Writer\ItemInterface;
use Symfony\Component\Validator\Constraints\Date;

/**
 * @ORM\Entity
 */
class Property implements ItemInterface
{
    /**
     * @ORM\Id
     * @ORM\Column(type="integer")
     * @ORM\GeneratedValue(strategy="AUTO")
     */
    protected $id;

    /**
     * @ORM\Column(type="integer")
     */
    protected $nbRooms;

    /**
     * @ORM\Column(type="integer")
     */
    protected $price;

    /**
     * @ORM\Column(type="date")
     */
    protected $publicationDate;

    /**
     * @ORM\Column(type="string", length=255)
     */
    protected $description;

    /**
     * @ORM\Column( type="array" )
     */
    protected $equipements;

    /**
     * @ORM\Column( type="array" )
     */
    protected $imagesPath;

    /**
     * @return mixed
     */
    public function getImagesPath()
    {
        return $this->imagesPath;
    }

    /**
     * @param mixed $imagesPath
     */
    public function setImagesPath($imagesPath)
    {
        $this->imagesPath = $imagesPath;
    }


    /**
     * @ORM\Column(type="string", length=255)
     */
    protected $location;

    /**
     * @ORM\Column(type="boolean")
     */
    protected $reported = false;

    /**
     * @ORM\Column(type="boolean")
     */
    protected $enabled = true;

    /**
     * @ORM\ManyToOne(targetEntity="UserBundle\Entity\User")
     */
    protected $host;

    /**
     * Property constructor.
     */
    public function __construct()
    {
        $this->publicationDate = new \DateTime;
    }

    /**
     * @return mixed
     */
    public function getId()
    {
        return $this->id;
    }

    /**
     * @param mixed $id
     */
    public function setId($id)
    {
        $this->id = $id;
    }

    /**
     * @return mixed
     */
    public function getNbRooms()
    {
        return $this->nbRooms;
    }

    /**
     * @param mixed $nbRooms
     */
    public function setNbRooms($nbRooms)
    {
        $this->nbRooms = $nbRooms;
    }

    /**
     * @return mixed
     */
    public function getPrice()
    {
        return $this->price;
    }

    /**
     * @param mixed $price
     */
    public function setPrice($price)
    {
        $this->price = $price;
    }

    /**
     * @return mixed
     */
    public function getPublicationDate()
    {
        return $this->publicationDate;
    }

    /**
     * @param mixed $publicationDate
     */
    public function setPublicationDate($publicationDate)
    {
        $this->publicationDate = $publicationDate;
    }

    /**
     * @return mixed
     */
    public function getDescription()
    {
        return $this->description;
    }

    /**
     * @param mixed $description
     */
    public function setDescription($description)
    {
        $this->description = $description;
    }

    /**
     * @return mixed
     */
    public function getEquipements()
    {
        return $this->equipements;
    }

    /**
     * @param mixed $equipements
     */
    public function setEquipements($equipements)
    {
        $this->equipements = $equipements;
    }

    /**
     * @return mixed
     */
    public function getReported()
    {
        return $this->reported;
    }

    /**
     * @param mixed $reported
     */
    public function setReported($reported)
    {
        $this->reported = $reported;
    }

    /**
     * @return mixed
     */
    public function getEnabled()
    {
        return $this->enabled;
    }

    /**
     * @param mixed $enabled
     */
    public function setEnabled($enabled)
    {
        $this->enabled = $enabled;
    }

    /**
     * @return mixed
     */
    public function getHost()
    {
        return $this->host;
    }

    /**
     * @param mixed $host
     */
    public function setHost($host)
    {
        $this->host = $host;
    }

    /**
     * @return mixed
     */
    public function getLocation()
    {
        return $this->location;
    }

    /**
     * @param mixed $location
     */
    public function setLocation($location)
    {
        $this->location = $location;
    }


    /**
     * This method returns feed item title.
     *
     *
     * @return string
     */
    public function getFeedItemTitle()
    {
        return "Property N° ".$this->getId()." Located in ".$this->getLocation();
    }

    /**
     * This method returns feed item description (or content).
     *
     *
     * @return string
     */
    public function getFeedItemDescription()
    {
        return "<b>Description: </b>".$this->getDescription()."<br><b>Price:</b>  "
                .$this->getPrice()." TND<br> <b>Equipement: </b>".implode(" - ", $this->getEquipements());

            ;
    }

    /**
     * This method returns feed item URL link.
     *
     *
     * @return string
     */
    public function getFeedItemLink()
    {
       return "http://localhost/PHPstormProjects/Host_n_Guest/web/app_dev.php/property/detailProperty/".$this->getId();
    }

    /**
     * This method returns item publication date.
     *
     *
     * @return \DateTime
     */
    public function getFeedItemPubDate()
    {
        return $this->getPublicationDate();
    }
}
